src/caf/distribute/gravity_model/multi_area.py

```diff
@@ -427,21 +530,24 @@ def calibrate(
         `caf.distribute.furness.doubly_constrained_furness()`
         `scipy.optimize.least_squares()`
         """
-        for dist in self.dists:
+        for dist in distributions:
             self.cost_function.validate_params(dist.function_params)
         self._validate_running_log(running_log_path)
         self._initialise_internal_params()
         return self._calibrate(  # type: ignore
```

Comment:
Because `self._calibrate` is only called here and this method doesn't contain much else other than a call to `_calibrate` I think it would probably be simpler to move all the code from `_calibrate` to here and remove the separate method.

I made some comments to clean-up the `_calibrate` method which I still think would be helpful.

Reply:
I think this is likely to change the API significantly, is this more a to do and new major verision number since it will be incompatible with previous versions

Reply:
wasn't the idea with this wrapper to hide the more complicated args to avoid confusion?